extensions/resource-deployment/src/ui/modelViewUtils.ts

```diff
@@ -665,8 +726,12 @@ function processHyperlinkedTextField(context: FieldContext): ReadOnlyFieldInputs
 
 function processEvaluatedTextField(context: FieldContext): ReadOnlyFieldInputs {
 	const readOnlyField = processReadonlyTextField(context, false /*allowEvaluation*/);
+	const onChangedEmitter = new vscode.EventEmitter<void>(); // Stub event emitter since this value won't change
```

Comment:
This might be ok for now, but value of EvaluatedTextField does change when the field that it depends on changes, it just does not change on user input. While we may not need this yet, but perhaps modifying the comment that we do not care about firing the event in that scenario so far will eliminate any surprises to a future developer